.changelog/3959.bugfix.md

```diff
@@ -0,0 +1 @@
+workload/governance: fix case with multiple equal active proposals
```

Comment:
Maybe mark this as internal as the issue is in E2E tests.

Reply:
Agreed, was thinking the same, just wasn't sure what the usual convention for these cases  :+1: